plugins/e131/messages/E131ConfigMessages.proto

```diff
@@ -14,7 +14,7 @@
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
  *
  * E131ConfigMessages.proto
- * Defines the protocol buffers used to configure the Artnet device
```

Comment:
@nils-van-zuijlen Sorry I've just spotted, do you fancy opening a new PR to correct this to E1.31 as it's actually a copy/paste error from then?

Reply:
But isn't it the filename ?

Reply:
Yes, the first one should be. I meant line 17 where it talks about using E131ConfigMessages.proto to configure the 'Artnet' device!